examples/update-on-async-rendering/adding-event-listeners-create-subscription.js

```diff
@@ -0,0 +1,33 @@
+import {createSubscription} from 'create-subscription';
+
+const Subscription = createSubscription({
+  getCurrentValue(sourceProp) {
+    // Return the current value of the subscription (sourceProp).
+    // highlight-next-line
```

Comment:
IMO the highlighting in this code sample is distracting and I can't tell why some lines are highlighted and not others. I'd just drop all the highlights in this file especially since it isn't really a before/after and ~all the lines are significant.

Reply:
Cool. I was on the fence about this to begin with.